src/components/MainMenu/MainMenu.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import "./MainMenu.css";
+import image1 from './img/IMAGE (7).png';
+import image2 from './img/IMAGE (8).png';
+import image3 from './img/IMAGE (9).png';
+import image4 from './img/IMAGE (10).png';
+import image5 from './img/IMAGE (11).png';
+import image6 from './img/IMAGE (12).png';
+
+
+const PhoneTooltip = () => {
```

Comment:
Only moving PhoneTooltip to a separate component is left

Did not want to move all the pictures, but it's fine, won't need them later anyway